src/main/scala/com/gu/stripeCustomerSourceUpdated/SourceUpdatedSteps.scala

```diff
@@ -60,25 +62,22 @@ object SourceUpdatedSteps extends Logging {
       ReturnWithResponse(unauthorized)
   } yield res
 
+  // cats does not have ListT so after removing scalaz it became flatMap + traverse
   def getPaymentMethodsToUpdate(
     requests: Requests
   )(customer: StripeCustomerId, source: StripeSourceId): ApiGatewayOp[List[PaymentMethodFields]] = {
     val zuoraQuerier = ZuoraQuery(requests)
-    (for {
-      // similar to AccountController.updateCard in members-data-api
-      paymentMethods <- ListT.apply[ApiGatewayOp, AccountPaymentMethodIds](
-        ZuoraQueryPaymentMethod.getPaymentMethodForStripeCustomer(zuoraQuerier)(customer, source)
-          .withLogging("getPaymentMethodForStripeCustomer")
-      )
-      account <- ListT[ApiGatewayOp, AccountSummary](
-        ZuoraGetAccountSummary(requests)(paymentMethods.accountId.value)
-          .toApiGatewayOp("ZuoraGetAccountSummary failed").withLogging("getAccountSummary").map(_.pure[List])
-      )
-      defaultPaymentMethods <- ListT[ApiGatewayOp, PaymentMethodFields](
-        findDefaultOrSkip(account.basicInfo.defaultPaymentMethod, paymentMethods.paymentMethods)
-          .toList.pure[ApiGatewayOp].withLogging("skipIfNotDefault")
-      )
-    } yield defaultPaymentMethods).run
+    ZuoraQueryPaymentMethod.getPaymentMethodForStripeCustomer(zuoraQuerier)(customer, source)
+      .flatMap {
+        _.flatTraverse { paymentMethods =>
+          ZuoraGetAccountSummary(requests)(paymentMethods.accountId.value).toApiGatewayOp("ZuoraGetAccountSummary failed").map(_.pure[List])
+            .flatMap {
+              _.flatTraverse { account =>
+                findDefaultOrSkip(account.basicInfo.defaultPaymentMethod, paymentMethods.paymentMethods).toList.pure[ApiGatewayOp].withLogging("findDefaultOrSkip")
+              }
+            }
```

Comment:
Again too many monads :) :
```suggestion
          ZuoraGetAccountSummary(requests)(paymentMethods.accountId.value).toApiGatewayOp("ZuoraGetAccountSummary failed")
            .flatMap { account =>
              findDefaultOrSkip(account.basicInfo.defaultPaymentMethod, paymentMethods.paymentMethods).toList.pure[ApiGatewayOp].withLogging("findDefaultOrSkip")
            }
```